### PR TITLE
Node week3

### PR DESCRIPTION
**Pull request by AliRezaHassanloo2731**
homework week-3

### CODE REVIEW

File: api/src/routers/meals.js [hunk 1]
```diff
@@ -0,0 +1,187 @@
+import express from "express";
+import knex from "../database_client.js";
+
+const router = express.Router();
+
+//api/meals?maxPrice=90
+
+router.get("/", async (req, res) => {
+  try {
+    let query = knex.select("*").from("Meal");
```
Comment (babak-f) [inline api/src/routers/meals.js:10]: How come this is not a const? :)

File: api/src/routers/meals.js [hunk 2]
```diff
@@ -0,0 +1,187 @@
...
+  try {
+    const sortDir = req.query.sortDir || "asc";
+    const meals = await knex("Meal").orderBy("price", sortDir);
+    res.json(meals);
+  } catch (err) {
+    res.status(500).json({ error: err.message });
+  }
+});
+
+// router.get("/", async (req, res) => {
```
Comment (babak-f) [inline api/src/routers/meals.js:111]: It's considered better style to not leave commented code in the commit

**Review by babak-f — COMMENTED** [review]
Good job ;)

File: api/src/routers/meals.js (hunk 2 above)
Comment (AliRezaHassanloo2731) [reply api/src/routers/meals.js:111]: thank you for feed back

**Pull request merged**